src/extension.ts

```diff
@@ -191,6 +191,16 @@ export function activate(context: vscode.ExtensionContext): void {
         })
     );
 
+    // uploadJSFromTS
+    context.subscriptions.push(
+        vscode.commands.registerCommand('extension.vscode-janus-debug.uploadJSFromTS', (param) => {
+            if (vscode.window.activeTextEditor) {
+                const _param = vscode.window.activeTextEditor.document;
+                commands.uploadJSFromTS(loginData, _param);
+            }
```

Comment:
Yes I know I've removed the underlines, at least in this part of my source code.